managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -3445,6 +3445,32 @@ public PositionImpl getNextValidPosition(final PositionImpl position) {
         return next;
     }
 
+    public PositionImpl getValidPositionAfterSkippedEntries(final PositionImpl position, int skippedEntryNum) {
+        PositionImpl skippedPosition;
+        try {
+            skippedPosition = getValidPositionInternal(position, skippedEntryNum);
+        } catch (NullPointerException e) {
```

Comment:
Catching NPE is always a bad practice in Java 

Please verify the cause of the NPE and add a null check

Reply:
OK , I will fix